buildah/tests/apply-task-hook.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cp ${TMPF} ${TMPF}.read
```

Comment:
Yeah, I agree it is a bit foreign but I am not really if there is better options, For the tests I need to add an registry but I don't want to change the shipped tasks  to have that internal registry since that's not something  the user usually wants, so I am adding these scripts that only runs while testing the tasks to add dynamically some more stuff into the default task.

I agree it's confusing and I have added more comments to explains what this those but I am open to way to improve this.

Reply:
Thanks! That helps.